parachain/primitives/core/Cargo.toml

```diff
@@ -8,7 +8,6 @@ edition = "2021"
 serde = { version = "1.0.137", optional = true, features = [ "derive" ] }
 codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false }
 scale-info = { version = "2.2.0", default-features = false, features = [ "derive" ] }
-enum-iterator = "0.6.0"
```

Comment:
Is this an unrelated, unused module?

Reply:
yup, we were only [using it for `ChannelId`](https://github.com/Snowfork/snowbridge/pull/753/files#diff-0a114baf0463c3e1f1e8522bf78e02c55ce5fb44fef902204ba4caf5f6e3b951L4), so it's a knock-on effect of removing `ChannelId`.